site/_docs/variables.md

```diff
@@ -130,6 +130,14 @@ following is a reference of the available data.
       </p></td>
     </tr>
     <tr>
+      <td><p><code>site.html_files</code></p></td>
+      <td><p>
+
+        A list of all static HTML files.
```

Comment:
I just copied what was in `html_pages`. It might be better if they both said something like that - make it clear that `html_pages` is a subset of `pages` and `html_files` is a subset of `static_files`. This wasn't clear to me from the current docs, though it's clear once you loop through them to see what they contain.